dbt/include/trino/macros/materializations/materialized_view.sql

```diff
@@ -0,0 +1,80 @@
+{% materialization trino_materialized_view, adapter="trino" %} 
+{%- set target_relation = this %}
+{%- set existing_relation = load_relation(this) -%}
+{%- set config_grace_period = config.require('grace_period') %}
+{%- set config_max_import_duration = config.require('max_import_duration') %}
+{%- set config_drop_any_existing = config.require('drop_any_existing') %}
+{%- set config_cron = config.get('cron') %}
+{%- set config_refresh_interval = config.get('refresh_interval') %}
+
+
+{{ run_hooks(pre_hooks) }}
+
+{% if existing_relation is none %}
+
+    {{ log("No existing view found, creating materialized view...", info=true) }}
+    {%- set build_sql = build_materialized_view(target_relation,config_grace_period,config_max_import_duration,config_cron,config_refresh_interval) %}
```

Comment:
Please use consistent formatting and add a space after every `,`.

Reply:
Done